tests/test_calculus.py

```diff
@@ -13,7 +13,7 @@ def test_differentiate():
 
     assert quickTest("x + 2y + 3z + 4", differentiate, 'x') == "1"
     assert quickTest("x + 2y + 3z + 4", differentiate, 'y') == "2.0"
-    assert quickTest("x + 2y + 3z + 4", differentiate, 'z') == "3.0"
+    # assert quickTest("x + 2y + 3z + 4", differentiate, 'z') == "3.0"
```

Comment:
Add a FIXME label describing the exception.
```suggestion
    # FIXME: <the problem>
    # assert quickTest("x + 2y + 3z + 4", differentiate, 'z') == "3.0"
```
Always try to do this whenever there is an exception or something doesn't work as expected.